### PR TITLE
Upped min_range and filter out points under that threshold

### PR DESCRIPTION
**Pull request by haidai**
For firefly to be able to map correctly since it has a shell on.

### CODE REVIEW

File: src/node.cpp [hunk 1]
```diff
@@ -85,19 +85,27 @@ void publish_scan(ros::Publisher *pub,
     if (!inverted) { // assumes scan window at the top
         for (size_t i = 0; i < node_count; i++) {
             float read_value = (float) nodes[i].distance_q2/4.0f/1000;
+            float range = read_value * distance_factor;
+            if (range < scan_msg.range_min)
+                range = std::numeric_limits<float>::infinity();
```
Comment (po1) [inline src/node.cpp:90]: I assume there is a scan filter somewhere to remove values over a threshold?

**Comment by po1** [comment]
lgtm

File: src/node.cpp (hunk 1 above)
Comment (haidai) [inline src/node.cpp:90]: yep

Comment (haidai) [inline src/node.cpp:90]: Well, it's more like either ignore infs or use then as proof that there's free space up until max range. We've set things to ignore right now b/c an rplidar inf isn't meaningful.

Comment (po1) [inline src/node.cpp:90]: as long as we don't mistakenly clear obstacles on the costmap, that's good enough for me

**Comment by po1** [comment]
has there been any testing done?

**Comment by haidai** [comment]
Yes, on firefly.

**Pull request merged**